cmd/node/node.go

```diff
@@ -675,6 +676,10 @@ func (app *App) initServices(ctx context.Context,
 	app.host.Register(activation.AtxProtocol, pubsub.ChainGossipHandler(syncHandler, atxDB.HandleGossipAtx))
 	app.host.Register(txs.IncomingTxProtocol, pubsub.ChainGossipHandler(syncHandler, txHandler.HandleGossipTransaction))
 	app.host.Register(activation.PoetProofProtocol, poetListener.HandlePoetProofMessage)
+	hareGossipHandler := rabbit.GetHareMsgHandler()
+	if hareGossipHandler != nil {
```

Comment:
this can be nil only because of SuperHare? i would rather exit with error or panic if hare is not registered

Reply:
yes. done